plugins/arElasticSearchPlugin/lib/arElasticSearchPlugin.class.php

```diff
@@ -468,6 +484,12 @@ public static function modelClassFromQubitObjectClass($className)
      */
     protected function initialize()
     {
+        if (sfConfig::get('app_diacritics')) {
+            $this->config['index']['configuration']['analysis']['char_filter']['diacritics_lowercase'] = $this->loadDiacriticsMappings();
+        }
+
+        $this->loadMappings();
```

Comment:
Why do we need this extra mapping load?

Reply:
this second mapping is for the mapping.yml, separate function than loadDiacriticsMapping, https://github.com/artefactual/atom/blob/35d1a58ed3a05c4aba4362ada9304809faaed497/plugins/arElasticSearchPlugin/lib/arElasticSearchPlugin.class.php#L118

Reply:
But why do we need to call it here? It's not assigning the returned value to anything and it will be called below through `$this->loadAndNormalizeMappings();` before the mappings are really needed.